test/SILOptimizer/inline_begin_apply.sil

```diff
@@ -53,45 +53,42 @@ unwind:
 // CHECK:  [[TEMP:%.*]] = alloc_stack $Indirect<SomeSubclass>
 // CHECK:  [[MK_IND:%.*]] = function_ref @make_indirect
 // CHECK:  apply [[MK_IND]]<SomeSubclass>([[TEMP]])
-// CHECK:  br bb3
+// CHECK:  destroy_addr [[TEMP]] : $*Indirect<SomeSubclass>
+// CHECK:  cond_br %0, bb3, bb5
```

Comment:
These changes to test cases are just about visitation order?

Reply:
No, I tried to ensure the visitation order was the same as before. This is a result of generalizing the fall-through to ReturnBB optimization. So, the inliner does a bit more optimization on the fly now, which just fell out of simplifying the code.